etools-dropdown-multi.js

```diff
@@ -1,3 +1,3 @@
-import {EtoolsDropdownMulti} from './EtoolsDropdownMulti.js';
+import {EtoolsDropdownMulti} from './src/etools-dropdown-multi';
 
-window.customElements.define(EtoolsDropdownMulti.is, EtoolsDropdownMulti);
```

Comment:
the 2 separate files - etools-dropdown-multi and EtoolsDropdownMulti are in order to implement this best practice:
https://open-wc.org/guides/developing-components/publishing/#do-export-side-effects-separately